### ISSUE TITLE
Add Nushell

### ISSUE THREAD

**Issue by jntrnr** [alilleybrinker/langs-in-rust#70]
You should add Nushell to the list too ☺️

**Comment by alilleybrinker** [comment]
Absolutely! I'll add it the next time I'm able to sit down and add a batch of languages. Thanks!

**Comment by alilleybrinker** [comment]
Added!

**Comment by jntrnr** [comment]
I think your automated task removed it again 😅

**Comment by alilleybrinker** [comment]
Ugh, you're right. Not sure how that happened. I'll have to audit the recent commits to see if anything else got removed, and put up some guardrails to avoid removing languages in the future.

Thanks for the catch!

**Comment by alerque** [comment]
This got fixed at some point, then re-broken yesterday by #100. I'll submit another fix, but part of the issue is the update script nukes anything it fails to download whether because the URL is outdated or there is a transitory network issue.
